basics_dev/gsoc.md

```diff
@@ -552,6 +552,28 @@ for more information and qubes-specific background.
  
 **Mentor**: [Marek Marczykowski-Górecki](/team/)
 
+### Porting Qubes to POWER9/PPC64
+
+**Project**: Porting Qubes to POWER9/PPC64
+
+**Brief explanation**: Qubes currently only supports the x86_64 CPU architecture. Xen currently has additional support for Power9/PPC64 processors, however work needs to be done to integrate this into the Qubes build process, as well as work in integrating this with the Qubes toolstack and security model. This may also be beneficial in simplifying the process of porting to other architectures. Some related discussion (porting to ppc64):
```

Comment:
Xen does _not_ have support for POWER9/PPC64 yet. It may be worth noting using KVM for this reason.